File: PrenotazioniVacanze/src/app/dati/alloggio.ts
export class Alloggio {
  
  destinazione :string;
  descrizione : string;
  dataInizio : Date;
  dataFine : Date;
  codAlloggio : number;
  camere : number;
  nBagni : number;
  nStanzaLetto : number;
  nLetti : number;
  maxPartecipanti : number;

  constructor ( codAlloggio : number, camere : number, nBagni : number, nStanzaLetto : number,
    nLetti : number, maxPartecipanti : number ,destinazione:string,descrizione:string,dataInizio:Date,
    dataFine:Date) {
      this.codAlloggio = codAlloggio;
      this.camere = camere;
      this.nBagni = nBagni;
      this.nLetti = nLetti;
      this.nStanzaLetto = nStanzaLetto;
      this.maxPartecipanti = maxPartecipanti;
      this.destinazione=destinazione;
      this.descrizione=descrizione;
      this.dataFine=dataFine;
      this.dataInizio=dataInizio;

  }

public getCodAlloggio() : number {
    return this.codAlloggio;
}

public getCamere() : number {
  return this.camere;
}
public setCamere(camere : number) {
  this.camere = camere;
}

public getNBagni() : number {
  return this.nBagni;
}
public setNBagni(nBagni : number) {
  this.nBagni = nBagni;
}

public getNStanzeLetto() : number {
  return this.nStanzaLetto;
}
public setNStanzeLetto(nStanzeLetto : number) {
  this.nStanzaLetto = nStanzeLetto;
}

public getNLetti() : number {
  return this.nLetti;
}
public setNLetti(nLetti : number) {
  this.nLetti = nLetti;
}

public getMaxPartecipanti() : number {
  return this.maxPartecipanti;
}
public setMaxPartecipanti( numero : number) {
  this.maxPartecipanti = numero;
}
public getDescrizione() : String {
  return this.descrizione;
}
public setDescrizione( descrizione : string) {
  this.descrizione = descrizione;
}
public getDestinazione() : String {
  return this.destinazione;
}
public setDestinazione( destinazione : string) {
  this.destinazione = destinazione;
}
public getDataInizio() : Date {
  return this.dataInizio;
}
public setDataInizio( dataInizio : Date) {
  this.dataInizio = dataInizio;
}

public getDataFine() : Date {
  return this.dataFine;
}
public setDataFine( dataFine : Date) {
  this.dataFine = dataFine;
}


}

File: PrenotazioniVacanzeSpring/src/main/java/com/example/PrenotazioniVacanzeSpring/Alloggio.java
package com.example.PrenotazioniVacanzeSpring;

import java.sql.Date;

public class Alloggio {

	  private String destinazione;
      private String descrizione;
	  private String pensione ;
	  private int nBagni ;
	  private int nStanzaLetto ;
	  private int nPartecipanti;
	  private Date dataInizio;
	  private Date dataFine;
      private String linkImg;
      private String titolo;
	  public Alloggio() {}
	public Alloggio(String destinazione, String descrizione, String pensione, int nBagni, int nStanzaLetto,
			int nLetti, Date dataInizio, Date dataFine,String linkImg,String titolo,int nPartecipanti) {
		super();
		this.destinazione = destinazione;
		this.descrizione = descrizione;
		this.pensione = pensione;
		this.nBagni = nBagni;
		this.nStanzaLetto = nStanzaLetto;
		this.dataInizio = dataInizio;
		this.dataFine = dataFine;
		this.linkImg=linkImg;
		this.titolo=titolo;
		this.nPartecipanti=nPartecipanti;
	}
	public String getDestinazione() {
		return destinazione;
	}
	public void setDestinazione(String destinazione) {
		this.destinazione = destinazione;
	}
	public String getDescrizione() {
		return descrizione;
	}
	public void setDescrizione(String descrizione) {
		this.descrizione = descrizione;
	}
	public String getCodAlloggio() {
		return pensione;
	}
	public void setCodAlloggio(String pensione) {
		this.pensione = pensione;
	}
	
	public int getnBagni() {
		return nBagni;
	}
	public void setnBagni(int nBagni) {
		this.nBagni = nBagni;
	}
	public int getnStanzaLetto() {
		return nStanzaLetto;
	}
	public void setnStanzaLetto(int nStanzaLetto) {
		this.nStanzaLetto = nStanzaLetto;
	}
	
	public Date getDataInizio() {
		return dataInizio;
	}
	public void setDataInizio(Date dataInizio) {
		this.dataInizio = dataInizio;
	}
	public Date getDataFine() {
		return dataFine;
	}
	public void setDataFine(Date dataFine) {
		this.dataFine = dataFine;
	}
	public String getLinkImg() {
		return linkImg;
	}
	public void setLinkImg(String linkImg) {
		this.linkImg = linkImg;
	}
	public String getTitolo() {
		return titolo;
	}
	public void setTitolo(String titolo) {
		this.titolo = titolo;
	}
	public String getPensione() {
		return pensione;
	}
	public void setPensione(String pensione) {
		this.pensione = pensione;
	}
	public int getnPartecipanti() {
		return nPartecipanti;
	}
	public void setnPartecipanti(int nPartecipanti) {
		this.nPartecipanti = nPartecipanti;
	}
	
	  
}
